src/background.js

```diff
@@ -13,25 +13,38 @@ chrome.runtime.onInstalled.addListener(function () {
     });
 });
 
-function sendVideoUrlToMetube(videoUrl, metubeUrl, format, callback) {
+function sendVideoUrlToMetube(videoUrl, metubeUrl, format, advancedSettings, callback) {
     console.log("Sending videoUrl=" + videoUrl + " to metubeUrl=" + metubeUrl);
 
     if (typeof callback !== 'function') {
         callback = function () {
         };
     }
 
+    let {hostname} = new URL(videoUrl)
+
+    let postData = {
+      "quality": "best",
+      "format": format,
+      "url": videoUrl,
+      'auto_start': !advancedSettings['disable_auto_start']
```

Comment:
may you add check for default settings. it's need for case when user not saved their options after updating extension.

```javascript
'auto_start': !advancedSettings['disable_auto_start'] ?? true
```


<img width="874" alt="screenshot-2024-03-22-22-38-5b27b937e2" src="https://github.com/Rpsl/metube-browser-extension/assets/265634/9e3ffec7-2631-4b5b-aef1-9c674225e484">

Reply:
damn didn't think about that